src/datasource.ts

```diff
@@ -355,6 +353,23 @@ export class OpenSearchDatasource extends DataSourceApi<OpenSearchQuery, OpenSea
   }
 
   testDatasource() {
+    if (!this.flavor || !valid(this.version)) {
+      return Promise.resolve({
+        status: 'error',
+        message: 'No version set',
+      });
+    }
+    // Elasticsearch versions after 7.10 are unsupported
+    if (this.flavor === Flavor.Elasticsearch && gte(this.version, '7.11.0')) {
+      return Promise.resolve({
+        status: 'error',
+        message:
+          'ElasticSearch version ' +
+          this.version +
+          ` is not supported by the OpenSearch plugin. Use the ElasticSearch plugin.`,
+      });
+    }
+    //https://grafana.com/docs/grafana/latest/datasources/elasticsearch/
```

Comment:
nice addition! Maybe we can share a link somehow?

Reply:
The commented out link was from when I was trying to add the link. I couldn't get it to render properly 😞, whatever calls `testDatasource` displayed the whole href block.